tanner/config.py

```diff
@@ -11,7 +11,8 @@
                    'REDIS': {'host': 'localhost', 'port': 6379, 'poolsize': 80, 'timeout': 1},
                    'EMULATORS': {'root_dir': '/opt/tanner'},
                    'SQLI': {'db_name': 'tanner.db'},
-                   'LOGGER': {'log_file': '/opt/tanner/tanner.log'},
+                   'DEBUG_LOGGER': {'log_file': '/opt/tanner/tanner.log'},
+                   'ERROR_LOGGER': {'log_file': '/opt/tanner/tanner.err'},
```

Comment:
I think making LOGGER as parent would make the code look better. Something like this
`'LOGGER': {'error_log_file': '/opt/tanner/tanner_errors.log', 'event_log_file': '/opt/tanner/tanner_events.log'}` in place of line 14 and 15. What do you say?